src/main/java/ConsoleOut/StringShow.java

```diff
@@ -0,0 +1,15 @@
+package ConsoleOut;
+
+public class StringShow {
+    void ShowString(String text, int delay, String color){
+        for(int i=0; i<text.length();i++){
```

Comment:
It can be replaced by tex.toCharArray().forEach(\<lambda-expression-with-logic-printing>), or normal for each on Char Array (for int loop is very complicating).